src/smolvm/images/published.py

```
@@ -232,43 +261,61 @@ def _manifest_row(preset: Preset, arch: Arch, vmm: Vmm, rootfs_sha256: str) -> P
 
 
 def _preset_rows(
-    preset: Preset, amd64_sha: str, arm64_sha: str
-) -> dict[tuple[Preset, Arch, Vmm], PublishedImage]:
-    """Generate all (preset, arch, vmm) manifest rows for a single preset."""
+    preset: Preset, amd64_sha: str, arm64_sha: str, *, os: Os = "ubuntu"
+) -> dict[ManifestKey, PublishedImage]:
+    """Generate all (preset, arch, vmm) manifest rows for a single preset.
+
+    Each call produces 6 rows (2 archs × 3 vmms) for one OS flavour.
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)